src/transformers/models/owlvit/configuration_owlvit.py

```diff
@@ -285,15 +285,15 @@ def __init__(
         return_dict=True,
         **kwargs
     ):
-        super().__init__(text_config=text_config, vision_config=vision_config, **kwargs)
+        super().__init__(**kwargs)
```

Comment:
We don't need to pass text/vision config to `super`, as we will set `self.text_config` and `self.vision_config` below